### ISSUE TITLE
Hidden operation installation

### ISSUE THREAD

**Issue by romw** [BOINC/boinc#1172]
**Reported by SCHHCC Tech on 4 Dec 42578826 06:59 UTC**
Reference: ticket #393 ?[public PC with admin account. Upon windows logoff; machine is 're-imaged'.[[BR]([BR]]

Environment:)]

Requirement: No public person is to have access to Boinc via the Boinc tray icon. Other obvious routes can be hidden; less obvious routes, oh well.[I have eliminated the MSCONFIG entry ...boincmgr.exe /a /s. This is replaced with a Task Scheduler enrty ...bioncmgr /a /m. Trigger is startup + 1 minute, running under a 2nd hidden admin account. This avoids the tray icon.[[BR]([BR]]

Approach:)]

results: Mostly works, however at times no processing is kick off. May well not be due to my approach.[Is boinctray.exe needed in MSCONFIG to launch under the active / logon admin account? Gossip says it is needed to honer the PC-in-use constraints.[[BR]([BR]]

Question-1:)]

Question-2: Is it or was it ever true that the install option to run in protected mode requires a non-admin account to host the session, and does that account have to be non-batch?[The approach of 'protected mode' is known to corrupt the gpu for other apps?[[BR]([BR]]

Question-3:)]

apologizes: As an old mainframe guy; I realize a trouble ticket system is not the place to ask questions.[[BR]]

bottom line: Provide a concise step by step doc to allow a 'hidden' approach to running Boinc.

Migrated-From: http://boinc.berkeley.edu/trac/ticket/1200

**Comment by romw** [comment]
**Commented by ChristianB on 8 Mar 42580321 06:12 UTC**
You're right this would be better on the boinc_dev [mailing list](http://boinc.berkeley.edu/email_lists.php) or the dev Forums. But let me try to answer some of your questions here:
If the PC gets re-Imaged it would be good to include the Client in this image and have the data directory on a server storage so the client can pick up where it left. I heard of this on the mailing list one time.

Some options for a mass deployment are here: [creating custom installers](http://boinc.berkeley.edu/wiki/Creating_custom_installers) (This is mainly for 6.x Clients so you may ask in the mailing list about some newer information

With a service install you an also restrict who can control the client on this machine (must be a member of the boinc_master group I think)

**Comment by ChristianBeer** [comment]
There are several ways to mass deploy BOINC nowadays. If there are still problems please open a new issue.
